### ISSUE TITLE
it's a bit surprising to default to MoveWindowToDesktopCenter in wip

### ISSUE THREAD

**Issue by JulienMaille** [wangwenx190/framelesshelper#108]
I'm not sure why such a feature is in the library in the first place to be honest ;)
At least the defaulf value should be not to recenter the window on startup, what do you think?

**Comment by wangwenx190** [comment]
Qt itself will put the window at desktop center on its first show, but after making it a native window, the initial position will always have some offset that will make the window shift some pixels to the bottom right corner of the screen, I'm not interested in finding what caused it but I think it doesn't look very good, so I manually move the window back to the desktop center before it become visible to the user for the first time.
Anyway, I think it's normal to put a window on the screen center on startup, it also looks good.
I can change the default behavior of framelesshelper of course, but I just don't understand why you don't need this behavior. Can you explain your use case a little more detailly?

**Comment by JulienMaille** [comment]
The reason is that I save the window position on close and restore it on next launch, which I believe is a standard way to go

**Comment by wangwenx190** [comment]
It's a reasonable use case indeed. In this case you just need to use the following option `Option::DontMoveWindowToDesktopCenter`
https://github.com/wangwenx190/framelesshelper/blob/bde9c85cd9bf0d408839f425a8218991e028d95e/include/FramelessHelper/Core/framelesshelpercore_global.h#L193
in here:
https://github.com/wangwenx190/framelesshelper/blob/bde9c85cd9bf0d408839f425a8218991e028d95e/examples/widget/widget.cpp#L40
You can combine multiple options using the bit-wise operator `|`.

**Comment by JulienMaille** [comment]
Yes that's I have been using but can you explain what this means?
`    UseStandardWindowLayout, // The standard window layout is a titlebar always on top and fill the window width.
`

**Comment by JulienMaille** [comment]
Oh I think I understand, you create the min/max/close buttons if this option isn't enabled, correct?

**Comment by wangwenx190** [comment]
The comments in the header may be misleading, really sorry, I'll update it tonight. Using that option will cause FramelessHelper create a homemade titlebar for you, and setup a window layout to put the titlebar inside it, so if your window has a layout already, it will mess up your own layout, such as in a `QMainWindow`.

**Comment by wangwenx190** [comment]
Most of the time you will not need that option, because you will use your own titlebar.
